ddtrace/constants.py

```diff
@@ -58,3 +58,29 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+APPSEC_BLOCKED_RESPONSE_HTML = """
```

Comment:
I wonder if these should be in `ddtrace/internal/constants.py` instead?

Not a strong opinion, feel free to resolve if you are ok with them being in the public API.

Reply:
Or better yet, `ddtrace.appsec._constants` (on next commit)

Reply:
Mmmm I think some users could be interested on checking where the default blocking response comes, but this being Python not being in the public API would not really put a stop to it.

Reply:
Moved in next commit.